lang/test/org/partiql/lang/eval/EvaluatingCompilerCastTest.kt

```diff
@@ -35,6 +35,6 @@ class EvaluatingCompilerCastTest : CastTestBase() {
     @ArgumentsSource(DateTimeConfiguredCastArguments::class)
     fun dateTimeConfiguredCast(dateTimeConfiguredCastCase: ConfiguredCastCase) =
         dateTimeConfiguredCastCase.assertCase(
-            ExpectedResultFormat.STRING
+            ExpectedResultFormat.PARTIQL
```

Comment:
While this is an improvement over string equality and is perfectly acceptable for `TIME` types, I might have considered `ExpectedResultFormat.ION` instead. See https://github.com/partiql/partiql-lang-kotlin/issues/560 for an explanation as to why none of the expected value formats in use today are truly ideal.  However, `ExpectedResultFormat.ION` is usually the most ideal one available today.